README.md

```diff
@@ -7,10 +7,7 @@ It serves to provide a minimum level of functionality to allow simple,
 repeatable demonstrations and end-to-end tests to be realized with as
 little supporting infrastructure as possible.
 
-It is not intended to be any kind of comprehensive example of the
-breadth of functionality exposed though the OpenAssetIO API.
-See the SampleAssetManager for a more concrete example of canonical
-manager behavior.
+It is not intended to be any kind of comprehensive example of the breadth of functionality exposed through the OpenAssetIO API. For a more detailed reference implementation, see [OpenAssetIO-MediaCreation](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation) or consult the [official documentation](https://openassetio.github.io/).
```

Comment:
The text looks good, thanks. But please can you wrap it to 72 chars as per the [coding style guidelines](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/contributing/CODING_STYLE.md#line-wrapping) (and consistency with the surrounding text).